/groups/behroozi/hbowden/cocoa/Cocoa/installation_scripts/create_link_all_projects.sh: line 55: /dev/tty: No such device or address
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 225, in <module>
    train_samples, train_data_vectors, train_sigma8 = get_data_vectors(params_train, comm, rank, label_train)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 186, in get_data_vectors
    local_params_list, local_data_vector_list, local_sigma8_list = get_local_data_vector_list(params_list,rank,label)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 159, in get_local_data_vector_list
    data_vector = cocoa_model.calculate_data_vector(params_list[i])
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/cocoa_model.py", line 46, in calculate_data_vector
    compute_success = component.check_cache_and_compute(want_derived=False,
TypeError: check_cache_and_compute() got an unexpected keyword argument 'H0'
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[45164,1],0]
  Exit code:    1
--------------------------------------------------------------------------
